hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/client/functional/TestMetadataUtilRLIandSIRecordGeneration.java

```diff
@@ -605,22 +596,22 @@ public void testReducedByKeysForRLIRecords() throws IOException {
       fail("Should not have reached here");
     } catch (Exception e) {
       // expected. no-op
-      assertTrue(e.getCause() instanceof HoodieIOException);
+      assertInstanceOf(HoodieIOException.class, e.getCause());
```

Comment:
If you're going to fix this syntax, it is better to migrate to `assertThrows` to simplify the setup

Reply:
Reverting, let's address syntax changes in another PR.

Reply:
Will revert all formatting changes manually.